timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/BatchPingableLeader.java

```diff
@@ -46,7 +49,13 @@ public interface BatchPingableLeader {
     @Path("ping")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
-    Set<Client> ping(Set<Client> clients);
+    @Handle(
```

Comment:
This resource is effectively unused. It's relevant if we ever continue dev-ing on TimeLock partitioning and/or if we consider enabling single-leader batch mode (Sourcegraph says we aren't as-of now).